### DISCUSSION TITLE
ISO 20022 XSD conversion to JSON

### DISCUSSION THREAD

**Discussion by ahkal01** [prowide/prowide-iso20022#58] [Q&A] [answered]
Hi,

I can see that the library converts actual MX payments files from XML to JSON.
Can it also convert ISO 20022 XSD files to JSON?

Thanks
Alain

**Comment by zubri** [comment, marked as answer]
The library model is created out of the XSD files. 
And supports all ISO 20022 types, not only payments.

Could you explain the use case to convert the actual schema (XSD) to JSON?

Regards
